app/sdk/modifiers/modifierEnemyMinionAttackWatchGainKeyword.coffee

```diff
@@ -7,16 +7,11 @@ ModifierRanged = require 'app/sdk/modifiers/modifierRanged'
 ModifierForcefield = require 'app/sdk/modifiers/modifierForcefield'
 
 class ModifierEnemyMinionAttackWatchGainKeyword extends ModifierEnemyMinionAttackWatch
-
 	type:"ModifierEnemyMinionAttackWatchGainKeyword"
 	@type:"ModifierEnemyMinionAttackWatchGainKeyword"
-
 	@modifierName:"ModifierEnemyMinionAttackWatchGainKeyword"
 	@description:"Whenever an enemy minion attacks, this minion gains a random keyword"
-
-	fxResource: ["FX.Modifiers.ModifierEnemyMinionAttackWatch"]
-	fxResource: ["FX.Modifiers.ModifierGenericBuff"]
-
+	fxResource: ["FX.Modifiers.ModifierEnemyMinionAttackWatch", "FX.Modifiers.ModifierGenericBuff"]
```

Comment:
This is a behavioral change: `fxResource` now includes both FX instead of immediately overwriting the previous value.